:memo: Added disposition and more text.

Ainali opened the pull request:
I thought it could use some improvement and this is a first stab.

Ainali committed:
:memo: Added disposition and more text.

lokal-profil commented on README.md:
```diff
@@ -1,11 +1,24 @@
 
-Requires [jquery.loadTemplate-1.4.5.min.js](https://github.com/codepb/jquery-template).
+## How to use
+First, add the filename of the image on Wikimedia Commons (without File:) and press Kolla!. Then add the URL where the image has been improperly used in the field **Användning**. If you want to be attributed in another way than what the tool could figure out from the data on Wikimedia Commons, edit the field **Erkännande**. The four radio buttons in the bottom are used to set the tone of the lang ...
```
Also works with the prefix, or with the whole url

lokal-profil replied:
Probably easier to read if broken down into a list

Ainali replied:
Ah, if it works with and without prefix, or with url I believe it would probably be easiest to drop my parenthesis. I agree that a list might be clearer. I'll rearrange that paragraph.

lokal-profil commented:
Tack för att du ger denna lite kärlek =)

Ainali committed:
Cleanup and list added

As @lokal-profil suggested I used a clearer language and turned it into a list which should be easier to read.
